new_platforms/tests/oetests_enclave/OETestTA.c

```diff
@@ -181,7 +181,7 @@ oe_result_t ecall_TestOEGetReportV2(uint32_t flags)
 
 oe_result_t ecall_TestOEGetTargetInfoV1(uint32_t flags)
 {
-    uint8_t report_buffer[1024];
+    uint8_t report_buffer[4096];
```

Comment:
4k is probably too much to put on the stack. Use dynamic allocation instead.

Reply:
Done.